tools/common.props

```diff
@@ -17,5 +17,11 @@
     <LangVersion>latest</LangVersion>
     <!-- We are usually using a preview SDK -->
     <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
+    <RepoRoot>$(MSBuildProjectDirectory)\..\..\</RepoRoot>
```

Comment:
Where does `MSBuildProjectDirectory` point to?

Reply:
The csproj that is being packed.

Reply:
I wouldn't depend on the naming convention of the project paths.  This should be `$(MSBuildThisFileDirectory)\..`